qcodes/instrument_drivers/tektronix/Keithley_2600.py

```diff
@@ -19,20 +19,26 @@ def __init__(self, name, address, channel, **kwargs):
         super().__init__(name, address, terminator='\n', **kwargs)
         self._channel = channel
 
-        self.add_parameter('volt', get_cmd='measure.v()',
-                           get_parser=float, set_cmd='source.levelv={:.12f}',
+        self.add_parameter('volt',
+                           get_cmd='measure.v()',
+                           get_parser=float,
+                           set_cmd='source.levelv={:.12f}',
                            label='Voltage',
                            unit='V')
-        self.add_parameter('curr', get_cmd='measure.i()',
-                           get_parser=float, set_cmd='source.leveli={:.12f}',
+        self.add_parameter('curr',
+                           get_cmd='measure.i()',
+                           get_parser=float,
+                           set_cmd='source.leveli={:.12f}',
                            label='Current',
                            unit='A')
         self.add_parameter('mode',
                            get_cmd='source.func',
+                           get_parser=float,
```

Comment:
hah, that's a bizarre return value for a discrete setting. All makes sense now...